src/components/interfaces.ts

```diff
@@ -16,3 +17,11 @@ export type Width = "auto" | "half" | "full";
 // used to help track of event payloads to remove at 1.0.0 – see https://github.com/Esri/calcite-components/issues/3781
 /* Note : should be removed before `1.0 */
 export type DeprecatedEventPayload = any;
+
+export enum KindIcons {
```

Comment:
Is there a better place for this? Previously `notice` was referencing this from the `alert` `interfaces.ts`.

Reply:
I thought the enum types were causing issues on the doc site?

This is the correct place for types. constants would go in a resources file in the folder.

Reply:
These aren't publicly exposed, so not an issue for doc site. But they are used by more than one component - it seemed weird that previously Notice was referencing this from the Alert `resources` file - maybe a `resources` file in the component root is better than this location (although this would be the only thing)?

Reply:
Yeah maybe resources is better.

Reply:
👍 added a shared `resources.ts` file to the `/component` folder root. Will merge after checks run.